model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -263,8 +304,11 @@ def _poll_perf_analyzer(self, process):
 
     def _auto_adjust_parameters(self, process):
         """
-        Use of the perf analyzer process
+        Attempt to update PA parameters based on the output
         """
+        # TODO-TMA-518 - how to handle?
```

Comment:
@matthewkotila I need to figure out how to handle this auto-adjusting for multi-model.

Reply:
fixed